lib/ngx/process.lua

```diff
@@ -0,0 +1,49 @@
+-- Copyright (C) Yichun Zhang (agentzh)
+
+
+local ffi = require 'ffi'
+local base = require "resty.core.base"
+local getfenv = getfenv
+
+local process_types = {
+    [base.FFI_PROCESS_SINGLE    ]  = "single process",
+    [base.FFI_PROCESS_MASTER    ]  = "master process",
+    [base.FFI_PROCESS_SIGNALLER ]  = "signaller process",
+    [base.FFI_PROCESS_WORKER    ]  = "worker process",
+    [base.FFI_PROCESS_HELPER    ]  = "helper process",
+    [base.FFI_PROCESS_PRIVILEGED]  = "privileged agent process",
```

Comment:
I think we can just omit the ` process` suffix. It's redundant info since they are all process types anyway.